### PR TITLE
Add "Cyberus Technology" to commercial support sites

### PR DESCRIPTION
**Pull request by jtraue**

### CODE REVIEW

File: community/commercial-support.tt [hunk 1]
```diff
@@ -187,6 +187,19 @@ development lifecycle.
         Turning operational issues into development tasks using Nix since 2013.
       </a>
     </li>
+
+    <li>
+      <a href="https://www.cyberus-technology.de/services?utm_source=nixos&utm_medium=referal&utm_campaign=nixos"/>
```
Comment (garbas) [inline community/commercial-support.tt:192]:
```suggestion
      <a href="https://www.cyberus-technology.de/services"/>
```

Comment (garbas) [reply community/commercial-support.tt:192]: Two fixes:
- html fix (a element is closed later on)
- lets include pure links without any tracking help

Comment (garbas) [reply community/commercial-support.tt]: Sorry to nag, but the SVG needs to be smaller right now the logo canvas is too big and is messing up the page.

Comment (jtraue) [reply community/commercial-support.tt]: Sorry for taking so long. I've updated the image. Can you please check again?

**Pull request merged**